package.json

```diff
@@ -40,6 +40,7 @@
     "@google-cloud/common": "^0.11.0",
     "continuation-local-storage": "^3.1.4",
     "gcp-metadata": "^0.1.0",
+    "is": "^3.2.0",
```

Comment:
I think we should use `is` consistently everywhere as that is what is used on `google-code-node` as well. This module doesn't use `lodash.isObject` anywhere.